cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java

```diff
@@ -307,7 +307,7 @@ private static boolean queryDatabaseMeta(String[] strHeadInfo, File file, Buffer
         ResultSet resultSet = statement.getResultSet();
         try {
           if (resultSet.next()) {
```

Comment:
can you add a comment to describe the structure of ResultSet?

Reply:
ok, I will add some description. Thank you for pointing this out.